### PR TITLE
AVRO-3275: Clarify specification on naming

### PR DESCRIPTION
**Pull request by opwvhk**
Mirror of PR apache/avro#1439

### CODE REVIEW

**Review by martin-g — APPROVED** [review]
Thank you, @opwvhk !
I will merge it once the upstream PR is merged!

File: content/en/docs/1.11.0/Specification/_index.md
Comment (RyanSkraba) [inline content/en/docs/1.11.0/Specification/_index.md:194]:
```suggestion
        "name": "Simple",
```

File: content/en/docs/1.11.0/Specification/_index.md
Comment (RyanSkraba) [inline content/en/docs/1.11.0/Specification/_index.md:196]:
```suggestion
        "doc": "A simple name (attribute) and a namespace (attribute); the fullname is 'explicit.Simple' (this is a different type than of the 'inheritNull' field).",
```

So nitpicky for consistency :D

**Review by RyanSkraba — APPROVED** [review]

**Comment by opwvhk** [comment]
As the changes are on the Avro website, this PR is no longer needed.

**Pull request closed without merging**
